db/migrate/20220326112137_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :Text
+      t.references :Author, foregin_key: { to_table: 'users' }, index:true
```

Comment:
- [ ] Kindly note that it's always a good practice to name all columns in a Postgres table in lowercase and separate words with `_`. example `posts_pounter` instead of `PostsCounter`, `author` instead of `Author` .... Kindly rename these table's column accordingly.